README.md

```diff
@@ -1,4 +1,29 @@
-# smarthome-localhomesdk
+# Local Home SDK Virtual Environment
+
+A mock Local Home Platform implemented as stubs for the [Local Home SDK](https://github.com/actions-on-google/local-home-sdk)
+
+## Usage
+
+1. Place bundled Node Local Fulfillment app into `home-app` folder.
+2. `npm install`
+3. `npm run test`
+
+This is a mock Local Home Platform that depends on the Local Home SDK and a compiled [Local Fulfillment App](https://github.com/actions-on-google/smart-home-local)
```

Comment:
Consider reworking to the instruction to show how to import the package and use it.

Reply:
Will do in separate PR

Reply:
#21